app/models/comment.rb

```diff
@@ -194,4 +194,16 @@ def publish
     self
   end
 
+  def self.receive_mail(message)
+    node_id = message.subject[/#([\d]+)/, 1] #This took out the node ID from the subject line
+    unless node_id.nil?
+      node = Node.find(node_id)
+      user = User.find_by(email: message.from.first)
+      if user.present? && node_id.present?
```

Comment:
hi @namangupta01 , is multiple replies from users processed correctly?

Reply:
yes! i guess so. Why do you think it doesn't?

Reply:
actually with 2nd and 3rd reply I got some extra info processed when I implemented in my proposal but solved later using another gem. If `reverse_markdown gem` processing correctly, then great!

Reply:
Yes, please coordinate to reduce redundant code here if you've both implemented similar processes!

Reply:
Yes using `reverse_markdown_gem` we are getting correct processed content.
Have a look!

![screen shot 2018-05-26 at 1 35 54 am](https://user-images.githubusercontent.com/20709985/40564289-c0312564-6085-11e8-9a98-93b55106e21d.png)

After 2 replies.

Reply:
Actually, my concern is this area--

![edit_image](https://user-images.githubusercontent.com/20878070/40565008-8539768e-6088-11e8-998a-46147fe3cd02.png)

What do you think about this ?